### DISCUSSION TITLE
How to see linked apps to github acount

### DISCUSSION THREAD

**Discussion by rolandogbue** [community/community#132092] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do i  see all the apps and accounts my github is linked to?

**Comment by Yewzir** [comment, marked as answer]
@rolandogbue 

### Apps

1. Go to your account **Settings**.
2. In the navigation pane on the left, click on **Applications**.
3. Check out the tabs **Installed GitHub Apps**, **Authorized GitHub Apps**, and **Authorized OAuth Apps**.

### Accounts

1. Go to your account **Settings**.
2. In the navigation pane on the left, click on **Emails**.

### Sessions

1. Go to your account **Settings**.
2.  In the navigation pane on the left, click on **Sessions**. 

🎉

Kind regards
Yewzir

**Reply by rolandogbue** [reply]
Thanks

**Comment by nadeeshaofficial1** [comment]
1. Sign in to GitHub.
2. Click your profile photo and select Settings.
3. In the left sidebar, click Applications.
4. Review Authorized OAuth Apps and Authorized GitHub Apps to see linked apps and manage permissions or revoke access if needed.

**Reply by rolandogbue** [reply]
Thanks
